src/IndentRainbow.Extension/Options/OptionsManager.cs

```diff
@@ -119,14 +125,15 @@ public static void SaveSettings(int indentSize, string fileExtensionsString, str
             settingsStore.SaveColors(colors);
             settingsStore.SaveOpacityMultiplier(opacityMultiplier);
             settingsStore.SaveHighlightingMode(highlightingmode);
+            settingsStore.SaveColorMode(colormode);
             settingsStore.SaveDetectErrorsFlag(detectError);
             settingsStore.SaveErrorColor(errorColor);
 
             OptionsManager.indentSize.Set(indentSize);
             OptionsManager.fileExtensionsString.Set(fileExtensionsString);
             fileExtensionsDictionary.Set(LanguageParser.CreateDictionaryFromString(fileExtensionsString));
             OptionsManager.colors.Set(colors);
-            brushes.Set(ColorParser.ConvertStringToBrushArray(colors, opacityMultiplier));
+            brushes.Set(ColorParser.ConvertStringToBrushArray(colors, opacityMultiplier, (int) colormode));
```

Comment:
You will also need to add `OptionsManager.colorMode.set(colormode)` since the UI only reads from those properties and is currently left in an outdated state.